examples/petsc/index.rst

```diff
@@ -141,10 +142,11 @@ where we have identified the :math:`2\times 2` contravariant metric tensor :math
 This expression can be simplified to avoid the explicit Moore-Penrose pseudo-inverse,
 
 .. math::
+   \begin{aligned}
    \mathbf g &= \left(\frac{\partial \overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}}^T \frac{\partial\overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}} \right)^{-1}_{(2\times 2)} \frac{\partial \overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}}_{(2\times3)}^T
    \frac{\partial \overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}}_{(3\times2)} \left(\frac{\partial \overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}}^T \frac{\partial\overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}} \right)^{-T}_{(2\times 2)}
-
    &= \left(\frac{\partial \overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}}^T \frac{\partial\overset{\circ}{\mathbf{x}}}{\partial \mathbf{X}} \right)^{-1}_{(2\times 2)}
+   \end{aligned}
```

Comment:
This had been a 2-line equation with the MathJax formatting, but it can fit on one line (without the awkward scrollbar) if we delete the two alignment characters `&`. I proposed that in 5528f7f4f5ad66f29dc5429008448152e09ed3c9, which you can choose to keep or drop.

Reply:
Okay, yes we can keep it as one-line. But at least it is not rendered as 
<img width="1407" alt="Screen Shot 2020-03-17 at 11 42 09 AM" src="https://user-images.githubusercontent.com/39932030/76921316-598b3d00-6893-11ea-89ce-dbb69dfff0e4.png">